### DISCUSSION TITLE
Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by Anishkagupta04** [recodehive/resume-pitch#102] [Q&A]
[ANISHKA_RESUME (7).pdf](https://github.com/user-attachments/files/17084481/ANISHKA_RESUME.7.pdf)
Name:Anishka Gupta
@sanjay-kv Sir kindly review my resume🙌

**Comment by sanjay-kv** [comment]
[ans.pdf](https://github.com/user-attachments/files/17089515/ans.pdf)

@Anishkagupta04 
[ATS Version of CV (29).docx](https://github.com/user-attachments/files/17089516/ATS.Version.of.CV.29.docx)
